src/nodes.coffee

```diff
@@ -1416,6 +1423,19 @@ exports.Call = class Call extends Base
       fragments.push @makeCode(' />')
     fragments
 
+  astType: ->
+    if @isNew
+      'NewExpression'
+    else
+      'CallExpression'
+
+  astProperties: ->
+    return
+      callee: @variable.ast()
+      arguments: arg.ast() for arg in @args
+      optional: !!@soak
+      implicit: !!@implicit
```

Comment:
`implicit` is the only extension to the standard `CallExpression`/`NewExpression` AST